requirements/docs.txt

```diff
@@ -0,0 +1 @@
+django
```

Comment:
👍 for the separate requirements to build the docs, from my experience it is not only less burden for RTD service but some build requirements can break the docs actually.

Reply:
Yep!